### ISSUE TITLE
Username Character Restrictions

### ISSUE THREAD

**Issue by Glacier150** [sergeant-savage/chatter#17]
Prevent users from adding spaces or other non-desirable characters to their username.

Also add a display warning about how many characters and which characters can be part of a username, so users aren't confused why a message isn't sending.

**Comment by sergeant-savage** [comment]
Ideally we want the username to have no spaces, correct? We might eventually move to making people sign in so we can store their messages, but that would mean unique nicknames or something like that. I guess what I mean is where do you see this change leading to in the future? @Glacier150

**Comment by Glacier150** [comment]
Yes, removing spaces in the username I think is good, and also removing all non-ASCII characters from both usernames and messages.
